### PR TITLE
Feat/redonewording

### PR DESCRIPTION
**Pull request by JacquesRW**
super cool looking

### CODE REVIEW

**Comment by NicholasG04** [comment]
#### 📝 Changed routes:


| [**/**](https://prom-rmzv6s7jd-nicholasg04.vercel.app/) |
|:-:|
|<a href="https://prom-rmzv6s7jd-nicholasg04.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/vDz5dciPmusmY_RU01UcdvQ0oXUAiEO2qBBA5h8eh_TTyEET3ffJ6j_HUkOA0ZE6zg28ifr-vqZymmGYXAV4bw.png">(view full size)</a> |







Commit 697b841f1213829155b6d65bcfc1202cdeb093eb (https://prom-rmzv6s7jd-nicholasg04.vercel.app).

File: src/pages/index.tsx [hunk 1]
```diff
@@ -22,8 +23,10 @@ const Home: NextPage = () => {
   return (
     <HomeLayout>
       <Header />
+      <Element name="cards">
```
Comment (NicholasG04) [inline src/pages/index.tsx:26]: Do we need this if you're just using scrollTo a pixel height?

**Comment by JacquesRW** [comment]
yes yes scrolly things

File: src/pages/index.tsx (hunk 1 above)
Comment (JacquesRW) [reply src/pages/index.tsx:26]: no

**Review by NicholasG04 — APPROVED** [review]
Thank you so much for this Jacques, great job.

**Pull request merged**